src/app/complex/complex-results/complex-navigator/complex-navigator.component.ts

```diff
@@ -33,136 +27,4 @@ export class ComplexNavigatorComponent implements OnInit {
   set components(value: Set<Interactor>) {
     this._components = value;
   }
-
-  public doesComplexHasSubcomplexes(complex): boolean {
```

Comment:
These methods were moved to `TableInteractorColumnComponent` and no longer needed here.